src/verifiers/openAttestationEthereumTokenRegistryMinted.ts

```diff
@@ -0,0 +1,46 @@
+import { getData, v2, v3, WrappedDocument } from "@govtechsg/open-attestation";
+import { isWrappedV3Document, VerificationFragmentType, Verifier } from "../types/core";
+import { verifyMinted } from "../issued/verify";
+import { getTokenRegistrySmartContract } from "../common/smartContract/documentToSmartContracts";
+
+const name = "OpenAttestationEthereumTokenRegistryMinted";
+const type: VerificationFragmentType = "DOCUMENT_STATUS";
+export const openAttestationEthereumTokenRegistryMinted: Verifier<
+  WrappedDocument<v2.OpenAttestationDocument> | WrappedDocument<v3.OpenAttestationDocument>
+> = {
+  skip: () => {
+    return Promise.resolve({
+      status: "SKIPPED",
+      type,
+      name,
+      message: `Document issuers doesn't have "tokenRegistry" property or ${v3.Method.TokenRegistry} method`
+    });
+  },
+  test: document => {
+    if (isWrappedV3Document(document)) {
+      const documentData = getData(document);
+      return documentData.proof.method === v3.Method.TokenRegistry;
+    }
+    const documentData = getData(document);
+    return documentData.issuers.every(issuer => "tokenRegistry" in issuer);
```

Comment:
For token registry, we should check of `any`/`some` instead of `every`.

The reason is that if it's using tokenRegistry it **MUST** be only one issuer, cannot mix it around with documentstore or other tokenregistry. 

In the verify function we can throw `INVALID` if it has been mixed

Reply:
done, returning error if more than one token registry or mix of methods